src/nupic/regions/PyRegion.cpp

```diff
@@ -431,12 +485,26 @@ void PyRegion::deserialize(BundleIO& bundle)
 
 void PyRegion::write(capnp::AnyPointer::Builder& proto) const
 {
-  NTA_THROW << "Unimplemented method PyRegion::write.";
+  initcreate_example();
```

Comment:
Pending response to https://groups.google.com/d/msg/capnproto/1dJ97vebYdw/-kPz-66iEgAJ.